pkg/plugin/aggregation/aggregator.go

```diff
@@ -217,5 +225,20 @@ func (a *Aggregator) handleResult(result *plugin.Result) error {
 	io.Copy(f, result.Body)
 	glog.Infof("wrote results to %v\n", resultsFile)
 
+	// If it's a tarball, extract it
+	if extension == ".tar.gz" {
+		f.Close()
+		err = tarx.Extract(resultsFile, path.Join(a.OutputDir, result.Path()), &tarx.ExtractOptions{})
+		if err != nil {
+			glog.Errorf("Could not extract tar file %v: %v", resultsFile, err)
+			return err
```

Comment:
I'd have to pop up, but what happens if there is an error in the aggregation?  Is it just reported and tar.gz the whole results?